RELEASE.md

```diff
@@ -4,45 +4,52 @@ There are three types of Renku releases: Sprint release, Hot/bug fix release and
 dependencies between them to a minimum and coordinated component releases
 should not usually be required (using API versioning, feature flags etc.).
 
-If a merged auto-PR that has not yet been included in a Renku release turns
-out to be faulty, it has to be reverted in the Renku repository so it is not
-accidentally included in a release. A new auto-PR with the fix can then be
-created. The sprint release DOES NOT wait for a fix.
+⚠️ If a component turns out to be faulty, please revert to the previous version
+as soon as possible to prevent accidentally including it in the next Renku release.
+The sprint release DOES NOT wait for a fix.
+If a regression made it into a release, remember to add an acceptance test to
+prevent this from happening again.
 
-If a regression made it into a release, the fix must add an acceptance test
-that would catch it, within reason.
+🕛 The deadline for this is Tuesday morning after the sprint. Please be on time!
 
-Each sprint one of the teams takes on the responsibility of pushing out the release and makes a PR with updates 
-relevant _and adapted_ to USERS and ADMINS in the `CHANGELOG.rst` file.  Before tagging a release, ask the 
-Product Manager to review the release notes for communication about user-facing changes. 
 
-### Process [YAT]
+### Tag a release [Product team]
+
+🛳️ The product team updates the `CHANGELOG.rst` file when needed to make it more
+user friendly and potentially highlight relevant features or disrupting changes.
+This process can start on Tuesday after 12:00.
+As soon as the release PR is merged, the product team tag a release and inform YAT
+for the rollout.
+
+
+### Deploy [YAT]
 
 * Roll out the release: create one PR per deployment in the `terraform-renku` repository to update the version and make any needed configuration changes. Important: if needed, add the `scheduled maintenance` label to help avoiding it being merged too early. Limited should be rolled out before Renkulab, as a canary release (see below).
 * Plan a maintenance, create it in statuspage for all Renku deployments managed by us. Coordinate with external Renku deployments admins (UniFR, SV).
-  * Limited should be rolled out ~1 week before the Renkulab rollout, to give a chance to find bugs early. E.g. Limited on Wednesday and Renkulab on Monday.
+* Limited should be rolled out ~1 week before the Renkulab rollout, to give a chance to find bugs early. E.g. Limited on Wednesday and Renkulab on Monday.
 * Make the tag in the `renku` repository, monitor the automatic process of chart publishing and deployment on staging.
 * Merge terraform-renku PR, monitor Flux upgrading Renku everywhere.
-* If relevant, notify users on Discourse, Twitter using visual tools and short sentences. Coordinate with the Product Manager.
+* Contact the Product team before rolling out on Renkulab (ideally one day in advance), so that they can highlight prominent changes on Discourse, Twitter, and other channels.

```

Comment:
I wonder if we need some mechanism to actually block merging until this is done. A label?

Reply:
I don't think we need to _block_ deployment on user announcements- it can be my job to make sure those announcements go out in a timely manner considering the planned release schedule.